dask_ml/_partial.py

```diff
@@ -120,19 +121,10 @@ def fit(
         }
     )
 
-    graphs = {x_name: x.__dask_graph__(), name: dsk}
-    if hasattr(y, "__dask_graph__"):
-        graphs[y_name] = y.__dask_graph__()
-
-    try:
-        from dask.highlevelgraph import HighLevelGraph
-
-        new_dsk = HighLevelGraph.merge(*graphs.values())
-    except ImportError:
```

Comment:
Note: this `ImportError` branch is no longer needed based on our current lower bound of supported `dask` versions

https://github.com/dask/dask-ml/blob/f250432bba9046cdca3b464cb6e8eede879d3284/setup.py#L13-L14